language/stdlib/transaction_scripts/rotate_dual_attestation_info.move

```diff
@@ -43,9 +43,16 @@ fun rotate_dual_attestation_info(account: &signer, new_url: vector<u8>, new_key:
     DualAttestation::rotate_compliance_public_key(account, new_key)
 }
 spec fun rotate_dual_attestation_info {
+    use 0x1::Errors;
+
     include DualAttestation::RotateBaseUrlAbortsIf;
     include DualAttestation::RotateBaseUrlEnsures;
     include DualAttestation::RotateCompliancePublicKeyAbortsIf;
     include DualAttestation::RotateCompliancePublicKeyEnsures;
+
+    aborts_with [check]
+        Errors::NOT_PUBLISHED,
+        Errors::INVALID_ARGUMENT;
+        Errors::INVALID_STATE; // TODO: due to AbortsIfNotOperating for LibraTimestamp::now_seconds().
```

Comment:
I think this is fine to leave out as well since the script prologue will abort if the system isn't operating.

Reply:
Removed this error code by adding a "requires" clause.